mp/src/game/client/hl2/hud_credits.cpp

```diff
@@ -103,7 +85,8 @@ class CHudCredits : public CHudElement, public Panel
     void PrepareLine(HFont hFont, char const *pchLine);
 
     CPanelAnimationVar(vgui::HFont, m_hTextFont, "TextFont", "Default");
-    CPanelAnimationVar(Color, m_TextColor, "TextColor", "FgColor");
+    CPanelAnimationVar(Color, m_TextColor, "TextColor", "White");
+    CPanelAnimationVar(Color, m_bgColor, "BgColor", "Black");
```

Comment:
Since this is no longer a HUD element, where would these get set? You should do `LoadControlSettings("resource/ui/CreditsDialog.res");` in the constructor, with the panel itself being the only entry. You'd then be able to set these colors in there.

Reply:
I also would recommend setting the scheme to something in the constructor, and making sure to use it throughout the rest of this file. You can create a CreditsScheme.res file that just bases off of the source scheme, look at ConsoleScheme.res for more info on that.

Reply:
Was thinking this actually though felt it was weird to expose credits stuff to res files. I suppose the names themselves are exposed in `scripts/credits.txt` so that makes sense.

Reply:
Well the goal is to allow people to completely customize the UI if they want, which this would significantly help them do so.